### PR TITLE
pass the dkeyserver addr as input params

### PR DESCRIPTION
**Pull request by wanghouqi**
remove hand code ip and port, change to passed in by the startup
parameter [-h] and [-p]

Signed-off-by: wanghouqi <wanghouqi@vip.qq.com>

### CODE REVIEW

File: dkeycache/App/main.cpp [hunk 1]
```diff
@@ -59,8 +59,27 @@ void cleanup()
...
+        if (argv[i] == host_key && i + 1 < argc && argv[i + 1][0] != '-') {
+            host_index = i + 1;
+        } else if (argv[i] == port_key && i + 1 < argc && argv[i + 1][0] != '-') {
+            port_index = i + 1;
+        }
+    }
+    if (host_index == -1 || port_index == -1)
+    {
+        printf("\nusage: ehsm-dkeycache -h 127.0.0.1 -p 8888\n\n");
+        return -1;
```
Comment (syan10) [inline dkeycache/App/main.cpp:79]: you can follow the enroll_app, don't use "-h" or "-p"
just directly run as "ehsm-dkeycache <dkeyserver_url>" and the dkeyserver_url="127.0.0.1:8888"

Comment (syan10) [reply dkeycache/App/main.cpp:79]: ignore it, seems you need to parse the input to get the IP and port

Comment (wanghouqi) [reply dkeycache/App/main.cpp]: It is fixed.please check again.

File: dkeycache/App/main.cpp [hunk 2]
```diff
@@ -57,10 +66,44 @@ void cleanup()
         delete g_la_server;
 }
 
+static void show_usage_and_exit(int code) {
+    printf("\nusage: ehsm-dkeycache -h 127.0.0.1 -p 8888\n\n");
+    exit(code);
```
Comment (syan10) [inline dkeycache/App/main.cpp:71]: "-h" -> "-i"

"-h" for help, to print the usage

File: dkeycache/App/main.cpp [hunk 3]
```diff
@@ -3,6 +3,8 @@
 #include <signal.h>
 
 #include <enclave_u.h>
+#include <getopt.h>
```
Comment (syan10) [inline dkeycache/App/main.cpp:6]: change the commit title as 
“
pass the dkeyserver addr as input params
”

File: dkeycache/App/main.cpp (hunk 2 above)
Comment (wanghouqi) [reply dkeycache/App/main.cpp:71]: ok,It will be change.

File: dkeycache/App/main.cpp (hunk 3 above)
Comment (wanghouqi) [reply dkeycache/App/main.cpp:6]: changed

**Review by yang8621 — APPROVED** [review]

**Pull request merged**